src/webhook/regular_user_validation.py

```diff
@@ -0,0 +1,55 @@
+from flask import request, Blueprint, Response
+import sys, traceback
+import json
+import os
+import prometheus_client
+from prometheus_client import Counter
+from prometheus_client.core import CollectorRegistry
+
+from webhook.request_helper import validate, responses
+
+bp = Blueprint("regular-user-webhook", __name__)
+
+# define what we track, declare Counter, how many times this route is accessed
+REGULAR_USER_DENIED = Counter('webhook_regular_user_denied', 'The total number of regular-user requests denied', labelnames=["request_kind","request_namespace","request_username","request_operation"])
```

Comment:
I think this is redundant with Kubernetes 1.16+ (OCP 4.3 is >=1.16), so we should [get them for free](https://kubernetes.io/docs/reference/access-authn-authz/extensible-admission-controllers/#monitoring-admission-webhooks)? Maybe we can trim them here?